data/assets/examples/rotation/staticrotation/euler.asset

```diff
@@ -0,0 +1,29 @@
+-- Euler Angles
+-- This asset creates a SceneGraphNode that only displays coordinate axes. The rotation of
```

Comment:
Add part about using Euler angles in the frist sentence, and att similar infomration to the other assets as well. 

Otherwise, it is hard to figure out at first glance how these examples are different.